src/ogreExporter.cpp

```diff
@@ -23,10 +23,10 @@
 - maintained by Bitgate, Inc. for the purpose of keeping Ogre compatible with the latest	-
 - technologies.																				-
 ---------------------------------------------------------------------------------------------
-- Copyright (c) 2011 MFG Baden-W�rttemberg, Innovation Agency for IT and media.             -
+- Copyright (c) 2011 MFG Baden-W�rttemberg, Innovation Agency for IT and media.             -
```

Comment:
What's up with the umlaut?

Reply:
> What's up with the umlaut?

Not sure. I kept reverting it back, but vscode or vstudio would change it when I saved. I didn't investigate which one was doing that.